players.yaml

```diff
@@ -0,0 +1,8 @@
+activePlayers:
+  - name:  ArchmageInc
+    points: 10
+  - name: jimmyhmiller
+    points: 0
+  - name: dilbert88 
+    points: 0
+inActivePlayers: []
```

Comment:
FYI, camelCase makes inActivePlayers a bit confusing to read. I'm not saying you should change it. I'm just making an observation.